zulipterminal/config/color.py

```diff
@@ -1,9 +1,7 @@
 """
-COLOR
------
-Contains color definitions or functions common across all themes.
-For further details on themefiles look at the theme contribution guide.
```

Comment:
Maybe lets add the "For further details..." line in the `developer-file-overview.md`. Linking to the theme contribution guide from the overview doc make sense.

Reply:
Are you suggesting to do something similar to [this](https://github.com/zulip/zulip-terminal/pull/1208/files#diff-0bb745d3e470b3a0c93da4298bceebbdf4d2f1f0cf8a7624c0f2f4e5991e5ac9R20)?

Reply:
Seems like Neil has a different opinion.
In general I feel the theme contributing guide is not accessible from anywhere unless you open the `themes` directory or `color.py`. (Is it linked from anywhere?) Maybe a link that directs interested users/developers from the README/FAQ or any other relevant documentation could help.

Adding a link might be a problem while parsing though.

Reply:
We can always document these folders or improve this process later, and of course other documentation linkage is great to add otherwise.

This issue/PR is about decentralizing the data in this file and making it easy to keep it in sync.

Reply:
Resolving this comment now I've opened #1246.